specs/agents/metadata.md

```diff
@@ -108,7 +107,9 @@ On Linux, the container ID and some of the Kubernetes metadata can be extracted
 
  If we match, then the basename is assumed to be a container ID.
 
-If the Kubernetes pod name is not the hostname, it can be overridden by the `KUBERNETES_POD_NAME` environment variable, using the [Downward API](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/). In a similar manner, you can inform the agent of the node name and namespace, using the environment variables `KUBERNETES_NODE_NAME` and `KUBERNETES_NAMESPACE`.
+Sometimes the `KUBERNETES_POD_NAME` is set using the [Downward API](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/),
+so we set the pod name to its value if it exists. Otherwise, we set the pod name to the hostname, because by default, Kubernetes will set the hostname to the pod name.
```

Comment:
```suggestion
so we set the pod name to its value if it exists. Otherwise, we set the pod name to the hostname, because that's the default value Kubernetes uses.
```